src/Faker/ORM/Doctrine/Populator.php

```diff
@@ -55,6 +68,9 @@ public function addEntity($entity, $number, $customColumnFormatters = array(), $
     /**
      * Populate the database using all the Entity classes previously added.
      *
+     * Please note that large amounts of data will result in more memory usage since the the Populator will return
+     * all newly created PKs after executing
```

Comment:
What do you think about this?

```suggestion
     * all newly created primary keys after executing.
```